libraries/Network/src/NetworkUdp.cpp

```diff
@@ -292,7 +292,10 @@ void NetworkUDP::flush() {}
 
 int NetworkUDP::parsePacket() {
   if (rx_buffer) {
-    return 0;
+    if (rx_buffer->full()) { // packet was not read yet
+      return rx_buffer->available();
```

Comment:
can you show a benefit from returning the previous packet len more than once? If code did not care about it the first time, why should it care after that?

Reply:
for the library it is better than destroying it then allocate the next then destroy it ...

Reply:
library will not destroy it. It will just return 0, because old packet is still there and new one was not received. The point of returning positive length is to signal that a new packet has been received. This happens only once per packet.

Reply:
but the PR is about not returning a zero if the packet was not read until the end